src/vim9expr.c

```diff
@@ -412,7 +412,7 @@ compile_class_object_index(cctx_T *cctx, char_u **arg, type_T *type)
 	}
 
 	// Could be a function reference: "obj.Func".
-	m_idx = object_method_idx(cl, name, len);
+	object_method_lookup(cl, name, len, &m_idx);
```

Comment:
I think as the return value is ignored, you need to use the `(void)` cast here.  Otherwise static analysis may complain.  This comment also applies to other place where the `object_method_lookup` and `class_method_lookup` are used.

Reply:
> I think as the return value is ignored, you need to use the `(void)` cast here. Otherwise static analysis may complain. This comment also applies to other place where the `object_method_lookup` and `class_method_lookup` are used.

In the first experiment, any usage of the `*_idx` functions was removed, with the idea that they would be removed; but that doesn't make sense.  The `*_idx` version are convenient and easily remembered with the naming convention.

With this, there is never a need to declare an integer and then ignore its value, both
- `int idx = xxx_idx(...)`
- `m = xxx_lookup(..., NULL)`

can be used. And don't need any cumbersome `(void)`.